USAGE_NOTES.md

```diff
@@ -1,5 +1,5 @@
 # USAGE NOTES
```

Comment:
I think we should rename the file to `migrating.md`

Reply:
I renamed to MIGRATION_NOTES.md. Is it better now?